js/bybit.js

```diff
@@ -5117,54 +5616,65 @@ module.exports = class bybit extends Exchange {
          * @method
          * @name bybit#transfer
          * @description transfer currency internally between wallets on the same account
-         * @see https://bybit-exchange.github.io/docs/account_asset/#t-createinternaltransfer
+         * @see https://bybit-exchange.github.io/docs/account_asset/v3/#t-createinternaltransfer
          * @param {string} code unified currency code
          * @param {float} amount amount to transfer
          * @param {string} fromAccount account to transfer from
          * @param {string} toAccount account to transfer to
          * @param {object} params extra parameters specific to the bybit api endpoint
-         * @param {string} params.transfer_id UUID, which is unique across the platform
+         * @param {string} params.transferId UUID, which is unique across the platform
          * @returns {object} a [transfer structure]{@link https://docs.ccxt.com/en/latest/manual.html#transfer-structure}
          */
         await this.loadMarkets ();
-        const transferId = this.safeString (params, 'transfer_id', this.uuid ());
+        const transferId = this.safeString (params, 'transferId', this.uuid ());
         const accountTypes = this.safeValue (this.options, 'accountsByType', {});
         const fromId = this.safeString (accountTypes, fromAccount, fromAccount);
         const toId = this.safeString (accountTypes, toAccount, toAccount);
         const currency = this.currency (code);
         const amountToPrecision = this.currencyToPrecision (code, amount);
-        const request = {
-            'transfer_id': transferId,
-            'from_account_type': fromId,
-            'to_account_type': toId,
-            'coin': currency['id'],
-            'amount': amountToPrecision,
-        };
-        const response = await this.privatePostAssetV1PrivateTransfer (this.extend (request, params));
+        let method = undefined;
+        [ method, params ] = this.handleOptionAndParams (params, 'transfer', 'method', 'privatePostAssetV1PrivateTransfer'); // v1 preferred atm, because it supports funding
+        let request = undefined;
+        if (method === 'privatePostAssetV3PrivateTransferInterTransfer') {
+            request = {
+                'transferId': transferId,
+                'fromAccountType': fromId,
+                'toAccountType': toId,
+                'coin': currency['id'],
+                'amount': amountToPrecision,
+            };
+        } else {
+            request = {
+                'transferId': transferId,
```

Comment:
it should be `transfer_id` here right?

Reply:
good catch, copy-pasted codes several times, and that become broken accidentaly. fixed.